frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -406,6 +406,83 @@ export const experimentLogic = kea<experimentLogicType>({
                     return Math.ceil(2 * Math.sqrt(2) * Math.sqrt(controlCountData))
                 },
         ],
+        bestConversionVariant: [
```

Comment:
We're starting to have a lot of these calculation values on the frontend (in addition to the ones already like trendCount, entrants, runningTime), is there anything that would be reasonable to move to the backend for calculation?

Reply:
would be nice to just have a `results_significant: true` in the api results 😸

Reply:
Fair question!

I wouldn't want to do this yet, specially when the calculation methodology can be in flux (and we may want to do more with these values on the fly - like the old slider) which makes it a bad fit for the backend.

However, once this is reasonably static, I think it can be a good idea to move things to the backend.

Reply:
ok! fair